src/sentry/static/sentry/app/constants/index.tsx

```diff
@@ -251,3 +250,9 @@ export const ORGANIZATION_FETCH_ERROR_TYPES = {
 export const CONFIG_DOCS_URL = 'https://docs.sentry.io/server/config/';
 export const DISCOVER2_DOCS_URL =
   'https://docs.sentry.io/performance-monitoring/discover-queries/';
+
+export const IS_CI = !!process.env.IS_CI;
+export const IS_PERCY = !!process.env.IS_PERCY;
+export const NODE_ENV = process.env.NODE_ENV;
+export const DISABLE_RR_WEB = !!process.env.DISABLE_RR_WEB;
+export const SPA_DSN = process.env.SPA_DSN;
```

Comment:
These will be accessible from the runtime application as well but with wonky values. Is that ok?

Reply:
Yeah, in that case it would be `undefined` which seems reasonable